src/renderer/viz/expressions/basic/array.js

```diff
@@ -51,12 +51,6 @@ export default class BaseArray extends BaseExpression {
 
         this.type = `${type}-array`;
         this.elems = elems;
-
```

Comment:
This looks interesting. I think that will be useful, for example when features have several properties for registering the same phenomenon in different dates

Reply:
:confused: what do you mean?

Reply:
Just that there could be cases in which a feature, for example a county, has three properties with population for 3 census dates (2000, 2005, 2010), and that now it could be used in an array, for some kind of visualizations. I think that non-normalized schema is quite common